ethcore/sync/Cargo.toml

```diff
@@ -4,26 +4,27 @@ name = "ethcore-sync"
 version = "1.12.0"
 license = "GPL-3.0"
 authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
 
 [lib]
 
 [dependencies]
 client-traits = { path = "../client-traits" }
-common-types = { path = "../types" }
+types = { package = "common-types", path = "../types" }
 enum_primitive = "0.1.1"
 ethcore-io = { path = "../../util/io" }
-ethcore-light = { path = "../light" }
-ethcore-network = { path = "../../util/network" }
-ethcore-network-devp2p = { path = "../../util/network-devp2p" }
+light = { package = "ethcore-light", path = "../light" }
+network = { package = "ethcore-network", path = "../../util/network" }
+devp2p = { package = "ethcore-network-devp2p", path = "../../util/network-devp2p" }
 ethcore-private-tx = { path = "../private-tx" }
 ethereum-types = "0.6.0"
 ethkey = { path = "../../accounts/ethkey" }
 fastmap = { path = "../../util/fastmap" }
 futures = "0.1"
-keccak-hash = "0.2.0"
+hash = { package = "keccak-hash", version = "0.2.0"}
```

Comment:
Here like above I prefer the full name because `hash` is such a commonly used word.